src/crashlytics/getIssueDetails.ts

```diff
@@ -11,18 +12,14 @@ const apiClient = new Client({
 });
 
 export async function getIssueDetails(appId: string, issueId: string): Promise<string> {
+  const requestProjectId = parseProjectNumber(appId);
   try {
-    const requestProjectNumber = parseProjectNumber(appId);
-    if (requestProjectNumber === undefined) {
-      throw new FirebaseError("Unable to get the projectId from the AppId.");
-    }
-
     const response = await apiClient.request<void, string>({
```

Comment:
Should we add a log here?

Reply:
Added debug logs to the missing tool calls.